picard/ui/options/about.py

```diff
@@ -44,10 +44,7 @@ def load(self):
         }
         args.update(versions.as_dict(i18n=True))
 
-        formats = []
-        for exts, name in supported_formats():
-            formats.extend(exts)
-        args["formats"] = ", ".join(formats)
+        args["formats"] = ", ".join(supported_extensions())
```

Comment:
I think we should drop the dot in front of each extension to shorten the display: `.mp3, .mp2, ...` to `mp3, mp2`

Also the text still says **Supported formats**, but we list extensions, and in practice we care more about **format** than **extension** (since recent changes). But i think it isn't something dramatic ;)